### DISCUSSION TITLE
Update Verbage in Settings

### DISCUSSION THREAD

**Discussion by calhix** [Eloston/ungoogled-chromium#2196] [General]
This isn't a big deal at all and it's just because I have OCD but what about updating the language used in the settings menu? There's been talk before in regards to making changes such as a page upon first boot which could be seen as a slight deviation into making this more independent from vanilla chromium and this may be somewhat similar in that way but this _is_ ungoogled. This would mean renaming "You and Google" and deleting the word google from the settings within that so it makes more sense. This also includes deleting the reference to Safe Browsing within Privacy and security and also deleting "Theme Open Chrome Web Store" under Appearance since it doesn't do anything. I'm unsure about the Google Translate option under Languages though, it seems like it's useless as well.

**Comment by calhix** [comment]
Here's what I was thinking:
You and Google becomes 'Your Profile'
     Sync and Google services becomes 'Autocomplete' (just delete the line below that states 'Other Google services')

Privacy and security
     Security
          Safe Browsing (protection from dangerous sites) and other security settings becomes 'Secure connections and other settings'
     May want to change verbage below 'Preload for faster browsing and searching' within Cookies and other site data since it mentions Google, not sure

**Reply by tomasz1986** [reply]
Making changes to those strings is likely not that easy because they're multilanguage, so do it properly, they would need to be edited (translated) accordingly in all languages, not just in English. Removing some of them completely should probably be quite simple though.

**Reply by calhix** [reply]
True, didn't think about that. But it's not possible to make these changes just for English though? Then wait for others to submit PRs for other languages?
